src/renderer/component/fileViewer/view.jsx

```diff
@@ -107,9 +147,42 @@ class FileViewer extends React.PureComponent<Props> {
 
   playContent() {
     const { play, uri } = this.props;
+
+    if (this.startedPlayingCb) {
+      this.startTime = Date.now();
+    }
+
     play(uri);
   }
 
+  logTimeToStart() {
+    const { claim } = this.props;
+
+    if (this.startTime) {
+      this.playTime = Date.now();
+      this.fireAnalyticsEvent(claim, this.startTime, this.playTime);
+    }
+  }
+
+  fireAnalyticsEvent = (claim, startTime, playTime) => {
```

Comment:
Why would we log the same parameters with every analytics event?
There doesn’t appear to be any event name associated as well, this could make things difficult to differentiate events.

Reply:
Not sure what you mean. We log the same parameters, but only once per file. The actual api call is here https://github.com/lbryio/lbry-desktop/blob/master/src/renderer/analytics.js#L47

Reply:
Whoops. You are right @tzarebczan 

I need to make sure it doesn't fire if the video hasn't started downloading. Will be easy to add.